forte/data/data_pack.py

```diff
@@ -26,15 +26,16 @@
     Set,
     Callable,
     Tuple,
+    cast,
 )
 from functools import partial
+
 from typing_inspect import get_origin
 from packaging.version import Version
 import numpy as np
 from sortedcontainers import SortedList
 
 from forte.common.exception import (
-    ProcessExecutionException,
```

Comment:
I don't think it is a good idea to change any existing behaviors, such as Exceptions thrown. If you want to change the exception, it should be a different PR.